app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1260,6 +1260,16 @@ class BrowserTabFragment :
     private fun onBrowserMenuButtonPressed() {
         contentScopeScripts.sendSubscriptionEvent(createBreakageReportingEventData())
         viewModel.onBrowserMenuClicked(isCustomTab = isActiveCustomTab())
+
+        lifecycleScope.launch {
+            webViewCompatTestHelper.onBrowserMenuButtonPressed(webView)
+        }
+    }
+
+    private fun onPageStarted() {
+        lifecycleScope.launch(dispatchers.main()) {
+            webViewCompatTestHelper.onPageStarted(webView)
+        }
```

Comment:
why specify `dispatcher` in one of these but not the other?we just let the dispatcher be handled internally and the caller doesn't have to care, IMO